### ISSUE TITLE
Cannot specify "target" property or specify non-standard entity_id when creating action in yaml.

### ISSUE THREAD

**Issue by test32443** [nielsfaber/alarmo#739]
### Alarmo Version

1.9.9

### HA Version

2023.5.4

### Bug description

Whenever I am trying to create a custom action or call tts.speak service, I want to be able to provide "target:" property so I can use new Piper TTS, something as in example below:

service: tts.speak
data:
  cache: true
  media_player_entity_id: media_player.vlc_telnet
  message: Test
target:
  entity_id: tts.piper

When I switch to YAML format and enter target property and then click on Save button or click "TO UI" and then switch back to YAML, the target: property is removed.

Alternatively, I could've used following formatting:

service: tts.speak
data:
  media_player_entity_id: media_player.vlc_telnet
  message: "Alarm: {{open_sensors}}."
  entity_id: tts.piper

I can test this config by clicking TRY IT button and even toggle between UI and YAML display, but I cannot save it

### Steps to reproduce

1. Click on Actions, create new notification. Under TASK group, click on "TO YAML" button. Paste following code in YAML:

service: tts.speak
data:
  media_player_entity_id: media_player.vlc_telnet
  message: "Alarm: {{open_sensors}}."
  entity_id: tts.piper

Populate remaining values. Click on SAVE button, observe that session notification is not saved.

2. Do the same, but use different YAML formatting:

service: tts.speak
data:
  cache: true
  media_player_entity_id: media_player.vlc_telnet
  message: Test
target:
  entity_id: tts.piper

Click on TO UI button and TO YAML button. Observe target: property disappearing

**Comment by XtremeOwnageDotCom** [comment]
Can confirm, 

Also, when editing in YAML, and making required changes, it will not save.

![image](https://github.com/nielsfaber/alarmo/assets/5262735/82422c80-9d4b-4376-a7c6-11434b94787b)

However, does test just fine.

**Comment by XtremeOwnageDotCom** [comment]
AS a work around, you can use a standard home assistant automation, until @nielsfaber returns from vacation... or wherever he is.

``` yaml
alias: Alarmo - Announce Alarm State
description: ""
trigger:
  - platform: state
    entity_id:
      - alarm_control_panel.alarmo
    attribute: arm_mode
condition: []
action:
  - service: tts.speak
    data:
      cache: true
      media_player_entity_id: media_player.security_event_media_players
      message: Alarm is {{states('alarm_control_panel.alarmo')}}
    target:
      entity_id: tts.piper
mode: single
```

**Comment by nielsfaber** [comment]
The `tts.*` entities are a new addition in HA since April.
So far, Alarmo only supports the `tts.*` services (combined with media_player entities).
I will be making changes to allow assigning the `tts` entities as well (both in UI and YAML modes) in the upcoming release.
The `media_player_entity_id` will have to be provided in YAML mode, since it is a parameter which is specific to the Piper integration.

Note that Alarmo does not use the `target` parameter. The `entity_id` should be provided in the `data` section instead.

@XtremeOwnageDotCom 
> Also, when editing in YAML, and making required changes, it will not save.

This is due to the entered entity_id being not allowed, I would expect an error message to show up, but it doesn't. I will look into this.

@XtremeOwnageDotCom 
> until @nielsfaber returns from vacation... or wherever he is.

Please keep the conversation polite and show some patience. 
I am trying to maintain this project as my limited free time allows, you're welcome to contribute code improvements to help.

**Comment by XtremeOwnageDotCom** [comment]
> > until @nielsfaber returns from vacation... or wherever he is.
> 
> Please keep the conversation polite and show some patience. I am trying to maintain this project as my limited free time allows, you're welcome to contribute code improvements to help.

No rudeness intended, my apologies if it came off that way. I have much appreciation for your projects around here, and use many of them.

**Comment by nielsfaber** [comment]
The change mentioned in https://github.com/nielsfaber/alarmo/issues/739#issuecomment-1605399191 is now released in v1.9.10 of alarmo.
@test32443  Please update and check if the issue if resolved for you.
If so, you can close the issue.
Otherwise, please let me know.

**Comment by test32443** [comment]
I can confirm that it works now - thank you for your fix. I would like to close this issue now.
